js/index.js

```diff
@@ -0,0 +1,81 @@
+var url;
+var content = document.getElementById("content");
+var search = document.getElementById("search");
+
+search.addEventListener("click", () => {
+    var searchText = document.getElementById("searchText").value;
+    url = `https://www.googleapis.com/books/v1/volumes?q=${searchText}&maxResults=40`;
+    if (searchText !== "" && searchText !== " ") {
+        getData(url);
+    } else {
+        window.alert("Please enter a search term that contains at least one alphanumeric character")
+    }
+})
+
+window.onload = () => {
+    url = "https://www.googleapis.com/books/v1/volumes?q=pride+prejudice&maxResults=40"
+    getData(url);
+}
+
+const getData = (url) => {
+    var request = new XMLHttpRequest();
+    request.open('GET', url);
+    request.onload = () => {
+        if (request.status >= 200 && request.status < 400) {
+            var data = JSON.parse(request.responseText);
+            var books = data.items;
+            var eligible = [];
+            for (x = 0; x < books.length; x++) {
+                if (books[x].volumeInfo.imageLinks != null && books[x].volumeInfo.authors != null && books[x].volumeInfo.industryIdentifiers != null) {
```

Comment:
Multiple `if/else` usually make the code difficult to read. Think of refactoring it or maybe see if you can use a `switch`

Reply:
it's a single if-else statement but has multiple conditions, and I think a switch statement will be longer. please advise appropriately